CHANGELOG.md

```
@@ -1,18 +1,2751 @@
+- Fix scrollIntoView function with offset argument provided
+
+## 3.8.0 (October 28, 2021)
+
+### Changed
+
+- Refactor focus styles to use `focus-visible` in all components
+- Use custom properties in Position component
+- Make transition classes work with position or cover classes on the same element
+- Refactor animation component
+
+### Changed
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)